README.md

```diff
@@ -1,92 +1,145 @@
-# fh.wdlR
-A repo containing a basic R package for using Cromwell with WDL workflows at Fred Hutch. (Contact Amy Paguirigan for help).
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+<!-- BEWARE: THIS CODE DOESN'T RUN - IT'S JUST PSEUDO CODE FOR NOW -->
 
 
-## Install from GitHub
-You will need the following packages installed first:
-```{r}
-install.packages(pkgs = c("httr", "jsonlite", "magrittr",
-                "dplyr", "ssh", "purrr", "paws", "tidyr"))
-```
 
-Then you can install the most recent version of `fh.wdlR` by:
+# rcromwell
+
+<!-- badges: start -->
+[![Project Status: Experimental – Useable, some support, not open to feedback, unstable API.](https://getwilds.github.io/badges/badges/experimental.svg)](https://getwilds.github.io/badges/#experimental)
+[![R-CMD-check](https://github.com/getwilds/rcromwell/actions/workflows/R-CMD-check.yaml/badge.svg?branch=dev)](https://github.com/getwilds/rcromwell/actions/workflows/R-CMD-check.yaml)
+<!-- badges: end -->
+
+Convenience Tools for Managing WDL Workflows via Cromwell
+
+## Installation
+
+You can install the development version of `rcromwell` from [GitHub](https://github.com/) with:
 
 ```r
-require(remotes)
-remotes::install_github('FredHutch/fh.wdlR')
+# install.packages("remotes")
+remotes::install_github("getwilds/rcromwell")
 ```
 
 Install a specific release version (in this case v1.0) by:
+
 ```r
-require(remotes)
-remotes::install_github('FredHutch/fh.wdlR@v1.0')
+remotes::install_github('getwilds/rcromwell@v1.0')
 ```
 
 ## Set up your Cromwell Server
-Use instructions over in the [diy-cromwell-server repo](https://github.com/FredHutch/diy-cromwell-server) to get the configuration files needed and some testing workflows.  
+
+Use instructions over in the [diy-cromwell-server repo](https://github.com/FredHutch/diy-cromwell-server) to get the configuration files needed and some testing workflows.
 
 
 ## Example workflow process
 
-```{r}
-## Set your Cromwell URL
-setCromwellURL(nodeAndPort = "gizmoXXX:20202")
+Set your Cromwell URL
+
+```r
+library(rcromwell)
+library(dplyr)
+cromwell_config("http://gizmoXXX:20202")
```

Comment:
Change to something vanilla: `cromwell_config("http://localhost:20202")` or some such nonsense.

Reply:
will do